darts/explainability/shap_explainer.py

```diff
@@ -580,6 +601,7 @@ def shap_explanations(
                                 :, :, self.target_dim * (h - 1) + t_idx
                             ]
                         )
+                        tmp_t.data = shap_explanation_tmp.data
```

Comment:
is this required?

Reply:
Yep, I believe it is. Reason being that in this particular if condition the shap.Explanation object is 'manually' constructed so the tmp_t dict doesn't contain the feature values (i.e., data) yet.